x-pack/legacy/plugins/monitoring/public/views/loading/index.js

```diff
@@ -25,15 +26,6 @@ uiRoutes
 
         initSetupModeState($scope, $injector);
 
-        const setupMode = getSetupModeState();
```

Comment:
Why was this moved?

Reply:
`'/elasticsearch/nodes'` route will do its own `monitoringClusters()` call through `routeInitProvider` and if that fails it will be stuck at a blank screen again. There is no point of sending it to the nodes route if the clusters call fails.

We are calling `monitoringClusters()` twice this way, but at least this way it has less chances of failing (after it has just succeeded). I figured this is fine for now, since this will go away once the setup becomes obsolete